hermes.podspec

```diff
@@ -29,26 +17,36 @@ Pod::Spec.new do |spec|
   spec.license     = { type: "MIT", file: "LICENSE" }
   spec.author      = "Facebook"
   spec.source      = { git: "https://github.com/facebook/hermes.git", tag: "v#{spec.version}" }
-  spec.platforms   = { :osx => "10.14" }
+  spec.platforms   = { :osx => "10.14", :ios => "10.0" }
```

Comment:
I think we should update this to something more accurate. What is the minimum that Hermes can run? With React Native being its primary consumer, I think we could align that requirement with what React Native supports.

Reply:
Yup, align with min supported target of RN 👍

Reply:
Turns out its already aligned: https://github.com/facebook/react-native/blob/master/React.podspec#L39